nexmo-spring-boot-autoconfigure/src/test/kotlin/com/vonage/starter/VonageAutoConfigurationBuilderTest.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2011-2019 Nexmo Inc
+ * Copyright (c) 2011-2019 Vonage Inc
```

Comment:
Technically for these headers, the date needs to be changed and the entity is Vonage Holdings Corp - not super particular on that though.

Reply:
i think this one is fine